server/src/auth.js

```diff
@@ -41,7 +41,7 @@ export const authorize = async (req, res, next) => {
 export const registerAuthRoutes = (app) => {
   app.get('/auth/login', (req, res) => {
     const oauthUrl = url.format({
-      protocol: req.protocol,
+      protocol: c.isDev ? 'http' : 'https',
```

Comment:
On Heroku, it was always `http`, so I changed it like this.
(It relates to `secure: !isDev,` couple lines below, where `isDev` is also used.)

Reply:
Do you mean on `Now` it was always `http`? Anyway, it looks okay. :+1:

Reply:
I think on `Now` it was correctly `https`, because it should not be working with `http`. But on Heroku it is `http` and this is easy fix.